workspace/app/services/dev-settings.svc.js

```diff
@@ -0,0 +1,120 @@
+// -------------------------------------------------------
+// Impac Workspace Settings
+// --------
+// Providing Impac! Angular defaults and configuration abilities.
+// -------------------------------------------------------
+angular.module('impacWorkspace').service('DevSettings', function ($q, ImpacRoutes, ImpacLinking, ImpacDeveloper, DevUser) {
+  var _self = this;
+
+  var DEFAULTS = {
+    // API Endpoints
+    mnoeUrl: {
+      host: '',
+      base: '/mnoe/jpi/v1'
+    },
+    impacUrl: {
+      host: 'http://localhost:4000',
+      base: '/api'
+    },
+    // Selected Organization - first organisations if unspecified
+    orgUid: '',
+    // -----------------------------------------------
+    // Kpis configurations
+    // -----------------------------------------------
+    // Change this to `true` to force kpi's to allow multiple watchables, instead of
+    // using the mno_hub kpis#discover action maps kpis to have single watchables per kpi.
+    multipleWatchableMode: false,
+    // Stub widget templates - add new widgets!
+    //------------------------------------------------
+    // Widget templates are stored on Maestrano API, stub your
+    // new widget templates here, and contact us for permenant additions or
+    // changes to existing widget templates.
+    widgetsTemplates: [
+      // Example widget template, please see documentation "How-To: Create a Widget" for available
+      // options, and importantly, note how the path & metadata.template attributes work.
+      // --
+      // {
+      //   path: 'accounts/assets_summary',
+      //   name: 'Assets / Liabilities summary',
+      //   metadata: { template: 'accounts/assets_liability_summary' },
+      //   desc: 'Compare Assets and Liabilities accounts',
+      //   icon: 'pie-chart',
+      //   width: 3
+      // },
+    ]
+  };
+
+  this._defaults = angular.copy(DEFAULTS);
+
+  this.defaults = function () {
+    return angular.copy(_self._defaults);
+  };
+
+  this.resetDefaults = function () {
+    return _self._defaults = angular.copy(DEFAULTS);
+  }
+
+  // NOTE: this will be refactored a lot when mnoe routes are added as default
```

Comment:
Indeed -> isn't it a good occasion to do it? :)
Looks like you've done 80% of the work below...